### PR TITLE
[Enhancement] fix show processlist

### LINKED ISSUE #9997: Finished query in sleep state for a very long time
### Steps to reproduce the behavior (Required)
1. CREATE TABLE '...'
2. INSERT INTO '....'
3. SELECT  '....'

Query any external table, such as tpcds-1g  query 85

> select  substr(r_reason_desc,1,20)        ,avg(ws_quantity)        ,avg(wr_refunded_cash)        ,avg(wr_fee)  from web_sales, web_returns, web_page, customer_demographics cd1,       customer_demographics cd2, customer_address, date_dim, reason   where ws_web_page_sk = wp_web_page_sk    and ws_item_sk = wr_item_sk    and ws_order_number = wr_order_number    and ws_sold_date_sk = d_date_sk and d_year = 2000    and cd1.cd_demo_sk = wr_refunded_cdemo_sk     and cd2.cd_demo_sk = wr_returning_cdemo_sk    and ca_address_sk = wr_refunded_addr_sk    and r_reason_sk = wr_reason_sk    and    (     (      cd1.cd_marital_status = 'M'      and      cd1.cd_marital_status = cd2.cd_marital_status      and      cd1.cd_education_status = 'Advanced Degree'      and       cd1.cd_education_status = cd2.cd_education_status      and      ws_sales_price between 100.00 and 150.00     )    or     (      cd1.cd_marital_status = 'S'      and      cd1.cd_marital_status = cd2.cd_marital_status      and      cd1.cd_education_status = 'College'       and      cd1.cd_education_status = cd2.cd_education_status      and      ws_sales_price between 50.00 and 100.00     )    or     (      cd1.cd_marital_status = 'W'      and      cd1.cd_marital_status = cd2.cd_marital_status      and      cd1.cd_education_status = '2 yr Degree'      and      cd1.cd_education_status = cd2.cd_education_status      and      ws_sales_price between 150.00 and 200.00     )    )    and    (     (      ca_country = 'United States'      and      ca_state in ('IN', 'OH', 'NJ')      and ws_net_profit between 100 and 200       )     or     (      ca_country = 'United States'      and      ca_state in ('WI', 'CT', 'KY')      and ws_net_profit between 150 and 300       )     or     (      ca_country = 'United States'      and      ca_state in ('LA', 'IA', 'AR')      and ws_net_profit between 50 and 250       )    ) group by r_reason_desc order by substr(r_reason_desc,1,20)         ,avg(ws_quantity)         ,avg(wr_refunded_cash)         ,avg(wr_fee) limit 100;

```
+------------------------------+------------------+-----------------------+-------------+
| substr(r_reason_desc, 1, 20) | avg(ws_quantity) | avg(wr_refunded_cash) | avg(wr_fee) |
+------------------------------+------------------+-----------------------+-------------+
| Gift exchange                |               76 |          464.36000000 |  8.62000000 |
| Not the product that         |               70 |          876.67000000 | 46.67000000 |
| Parts missing                |                7 |          129.42000000 | 38.65000000 |
| reason 23                    |               47 |          734.61000000 |  6.17000000 |
| reason 25                    |                5 |           48.94000000 | 53.14500000 |
| reason 28                    |                8 |          306.20000000 | 37.06000000 |
+------------------------------+------------------+-----------------------+-------------+
```

### Expected behavior (Required)

When query is finished, there is no query in `show processlist`

### Real behavior (Required)

HQRZ8FMK3Q

### StarRocks version (Required)
 - You can get the StarRocks version by executing SQL `select current_version()`

```
MySQL [default_catalog.tpcds_1g_orc_zlib]> select current_version();
+-------------------+
| current_version() |
+-------------------+
| ZHANGYAN 9646a4c  |
+-------------------+
1 row in set (0.03 sec)
```

### PR DESCRIPTION
**Pull request by dirtysalt**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [x] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #9997

## Problem Summary(Required) ：

This PR is to:
1. show `state` of threads
2. fix dashboard.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

File: fe/fe-core/src/main/java/com/starrocks/http/action/SessionAction.java [hunk 1]
```diff
@@ -44,6 +44,7 @@ public class SessionAction extends WebBaseAction {
         SESSION_TABLE_HEADER.add("Cluster");
         SESSION_TABLE_HEADER.add("Db");
         SESSION_TABLE_HEADER.add("Command");
+        SESSION_TABLE_HEADER.add("ConnectionStartTime");
```
Comment (mofeiatwork) [inline fe/fe-core/src/main/java/com/starrocks/http/action/SessionAction.java:47]: Do we need `CommandStarTime`?

Comment (dirtysalt) [reply fe/fe-core/src/main/java/com/starrocks/http/action/SessionAction.java:47]: it's introduced in this PR. https://github.com/StarRocks/starrocks/pull/5590

But to be consistent, we have to show it on http dashboard too, like here.

image

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 4 / 11 (36.36%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/qe/QueryState.java|0|1|00.00%|[108]|
|:large_blue_circle:|com/starrocks/http/action/SessionAction.java|1|3|33.33%|[76, 77]|
|:large_blue_circle:|com/starrocks/qe/ShowExecutor.java|2|5|40.00%|[390, 391, 1622]|
|:large_blue_circle:|com/starrocks/qe/ConnectContext.java|1|2|50.00%|[570]|

**Comment by dirtysalt** [comment]
run starrocks_fe_unittest

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 4 / 7 (57.14%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/qe/QueryState.java|0|1|00.00%|[108]|
|:large_blue_circle:|com/starrocks/qe/ConnectContext.java|1|2|50.00%|[570]|
|:large_blue_circle:|com/starrocks/qe/ShowExecutor.java|2|3|66.67%|[1620]|
|:large_blue_circle:|com/starrocks/http/action/SessionAction.java|1|1|100.00%|[]|

**Review by Youngwb — APPROVED** [review]

**Review by DorianZheng — APPROVED** [review]
LGTM

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Review by caneGuy — APPROVED** [review]

**Comment by dirtysalt** [comment]
run starrocks_admit_test

**Comment by kangkaisen** [comment]
You should add a test case

**Comment by dirtysalt** [comment]
> You should add a test case

It's an enhancement: to add `state` in the output of `show processlist`. and there is a ut has covered that, I have to fix that ut to get passed.

**Pull request merged**